targets/mimxrt1062-teensy40.ld

```diff
@@ -8,7 +8,7 @@ MEMORY
 
 ENTRY(Reset_Handler);
 
-_stack_size = 128K;
+_stack_size = 4K;
```

Comment:
Looks like a reasonable change, but seems unrelated to USB code?

Reply:
Indeed, unrelated, I think this was me trying to resolve an issue that was ultimately caused by unexpected heap alloc, indirectly from interrupt. Also remembered you pointing out my misunderstanding of the variable.

Similarly, this change also part of the PR wasn't required for USB, but eliminates a heap alloc with the chip's runtime package:
https://github.com/ardnew/tinygo/blob/a6fe62faf8db13e6e9beccdbbf2e7326d644bcfc/src/device/nxp/mimxrt1062_mpu.go#L219-L279